schemas/container.json

```diff
@@ -251,13 +278,19 @@
         "environment": {
           "type": "string",
           "minLength": 1
+        },
+        "environmentId": {
+          "type": "string",
+          "pattern": "EN[a-zA-Z0-9]{32}",
+          "minLength": 1
```

Comment:
This is also another ticket we added while we were at it, not related to the rest of the work.